From 41b25549f080ebac1269689f942f722368ed28b1 Mon Sep 17 00:00:00 2001
From: Pawel Baldysiak <pawel.baldysiak@intel.com>
Date: Thu, 28 Sep 2017 14:41:13 +0200
Subject: [PATCH 07/12] Grow: fix switching on PPL during recovery

If raid memeber is not in sync - it is skipped during
enablement of PPL. This is not correct, since the drive that
we are currently recovering to does not have ppl_size and ppl_sector
properly set in sysfs.
Remove this skipping, so all drives are updated during turning on the PPL.

Signed-off-by: Pawel Baldysiak <pawel.baldysiak@intel.com>
Signed-off-by: Jes Sorensen <jsorensen@fb.com>
---
 Grow.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Grow.c b/Grow.c
index bab1eec..1149753 100644
--- a/Grow.c
+++ b/Grow.c
@@ -637,9 +637,6 @@ int Grow_consistency_policy(char *devname, int fd, struct context *c, struct sha
 			int dfd;
 			char *devpath;
 
-			if ((sd->disk.state & (1 << MD_DISK_SYNC)) == 0)
-				continue;
-
 			devpath = map_dev(sd->disk.major, sd->disk.minor, 0);
 			dfd = dev_open(devpath, O_RDWR);
 			if (dfd < 0) {
-- 
2.7.4

